libs/libtins/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2006-2017 OpenWrt.org
```

Comment:
Don't assign copyright to OpenWrt unless you're affiliated. Assign it to yourself or leave it out completely.